untitled/src/test/java/IssueTest.java

```diff
@@ -0,0 +1,25 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class IssueTest extends BaseTest {
+    @Test
+    public void checkLoginIsSuccessful() {
+        HomePage home = new HomePage(driver);
+        home.goToLoginPage().loginSuccessful("mari444ka1991@gmail.com", "More2023");
+        MainPage mainPage = new MainPage(driver);
+        mainPage.getLogoOnTheMainPage();
+        Assertions.assertTrue(mainPage.getLogoOnTheMainPage().isDisplayed());
+        mainPage.goToPersonalLogo().click();
+        mainPage.getYourRepositories().click();
+        RepositoriesPage repositoriesPage = new RepositoriesPage(driver);
+        repositoriesPage.getRepository().click();
+        IssuesPage issuesPage = new IssuesPage(driver);
+        issuesPage.getIssuesTab().click();
+        issuesPage.getOpenIssue().click();
+        issuesPage.typeIssue("Issue", "SomeBody in issue");
+        issuesPage.getEditTextButton().click();
+        issuesPage.typeTextToUpdateIssue("New Text", "Some comment");
```

Comment:
it's a good idea to separate your test for 2: 1 test for a new issue creation checking and 1 test - for an issue that can be updated